src/tds/bulk.c

```diff
@@ -760,16 +761,28 @@ tds7_bcp_send_colmetadata(TDSSOCKET *tds, TDSBCPINFO *bcpinfo)
 		 * different from BCP format
 		 */
 		if (is_blob_type(bcpcol->on_server.column_type)) {
-			/* FIXME support multibyte string */
-			len = tds_dstr_len(&bcpinfo->tablename);
-			TDS_PUT_SMALLINT(tds, len);
-			tds_put_string(tds, tds_dstr_cstr(&bcpinfo->tablename), len);
+			converted_name = tds_convert_string(tds, tds->conn->char_convs[client2ucs2], tds_dstr_cstr(&bcpinfo->tablename), (int)tds_dstr_len(&bcpinfo->tablename), &converted_len);
+			if (!converted_name) {
+				return TDS_FAIL;
```

Comment:
```
TDSRET
tds_bcp_start(TDSSOCKET *tds, TDSBCPINFO *bcpinfo)
{
	...

	if (IS_TDS7_PLUS(tds->conn))
		tds7_bcp_send_colmetadata(tds, bcpinfo);
	
	return TDS_SUCCESS;
}
```
`tds_bcp_start` when calling `tds7_bcp_send_colmetadata` does not check for errors.
Should I fix this?

Reply:
Good idea, not a regression here. Even better to write a better test for BCP and no-ascii encoding. But still not a regression here.